src/pages/workspace/WorkspacePageWithSections.tsx

```diff
@@ -143,7 +143,7 @@ function WorkspacePageWithSections({
             return true;
         }
 
-        return (!isEmptyObject(policy) && !PolicyUtils.isPolicyAdmin(policy) && !shouldShowNonAdmin) || PolicyUtils.isPendingDeletePolicy(policy);
+        return !isEmptyObject(policy) && !PolicyUtils.isPolicyAdmin(policy) && !shouldShowNonAdmin;
```

Comment:
This change will cause the regression

1. Go to the workspace that we going to delete
2. Press Reimbursements > Copy the URL
3. Press workspace Profile > Press Delete
4. Go to any chat paste the workspace reimbursements URL and send
5. Press the link, it will show the reimbursements page and we can edit the Rate

https://github.com/Expensify/App/assets/35371050/bd24804a-0d63-4814-9416-e9ac52c738c6

Reply:
To understand clearly the output I am attaching staging result.

https://github.com/Expensify/App/assets/35371050/9b56df9f-d77c-4b1a-a685-ba8b04741c92